packages/react-ui/internal/CustomComboBox/ComboBoxView.tsx

```diff
@@ -372,23 +359,13 @@ export class ComboBoxView<T> extends React.Component<ComboBoxViewProps<T>, Combo
     );
   }
 
-  private handleFocusMobile = () => {
-    this.setState({
-      isMobileOpened: true,
-    });
-
-    if (this.mobileInput) {
-      this.mobileInput.focus();
-    }
-  };
-
   private handleItemSelect = (item: T) => {
     if (this.props.onValueChange) {
       this.props.onValueChange(item);
     }
 
     if (this.isMobileLayout) {
-      this.handleCloseMobile();
+      this.props.onMobileClose && this.props.onMobileClose();
```

Comment:
Лаконичнее использовать [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).

```suggestion
      this.props.onMobileClose?.();
```

Reply:
Изменил на оператор.